openjdk/excludes/vendors/alibaba/ProblemList_openjdk11.txt

```diff
@@ -0,0 +1,122 @@
+############################################################################
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#############################################################################
+
+# hotspot_all
+
+############################################################################
+
+# hotspot_jre
+
+############################################################################
+
+# jdk_awt
+
+############################################################################
+
+# jdk_beans
+
+############################################################################
+
+# jdk_build
+
+############################################################################
+
+# jdk_compiler
+
+############################################################################
+
+
+# jdk_lang
+java/lang/management/MemoryMXBean/MemoryTestAllGC.sh https://github.com/adoptium/aqa-tests/issues/3731 generic-all
```

Comment:
I have made a mistake for the testcase name, the fail testcase is MemoryTestZGC.sh but not MemoryTestAllGC.sh

Reply:
```suggestion
java/lang/management/MemoryMXBean/MemoryTestZGC.sh https://github.com/adoptium/aqa-tests/issues/3731 generic-all
```